### ISSUE TITLE
Documentation issue: Labs

### ISSUE THREAD

**Issue by farcaller** [bazelbuild/rules_nodejs#2427]
Following the labs example for protos:

```python
load("@npm//@bazel/typescript:index.bzl", "ts_library", "ts_proto_library")
```

results in 

```
Error: file '@npm//@bazel/typescript:index.bzl' does not contain symbol 'ts_proto_library'
```

**Comment by alexeagle** [comment]
where did you find that wrong documentation? need a link pls

**Comment by farcaller** [comment]
Started a PR with a fix (#2434). Any advice on how to re-gen the html file there?

**Comment by mattem** [comment]
Fix by publish of doc site in 3.1.0
